VAMobile/src/screens/HealthScreen/TravelPay/TravelPayClaims/TravelPayClaimsList.tsx

```diff
@@ -13,36 +13,157 @@ import {
   Pagination,
   PaginationProps,
   TextLine,
+  TextView,
+  VAModalPicker,
 } from 'components'
 import { Events } from 'constants/analytics'
+import { DEFAULT_PAGE_SIZE } from 'constants/common'
 import { NAMESPACE } from 'constants/namespaces'
+import { TimeFrameType, TimeFrameTypeConstants } from 'constants/timeframes'
 import { logAnalyticsEvent } from 'utils/analytics'
 import getEnv from 'utils/env'
-import { getFormattedDateOrTimeWithFormatOption, getFormattedTimeForTimeZone } from 'utils/formattingUtils'
+import {
+  getFormattedDate,
+  getFormattedDateOrTimeWithFormatOption,
+  getFormattedTimeForTimeZone,
+} from 'utils/formattingUtils'
 import { useRouteNavigation, useTheme } from 'utils/hooks'
+import { createTimeFrameDateRangeMap } from 'utils/travelPay'
 
 const { LINK_URL_TRAVEL_PAY_WEB_DETAILS } = getEnv()
 
+const getMMMyyyy = (date: DateTime): string => {
+  return getFormattedDate(date.toISO(), 'MMM yyyy')
+}
+
+const getDateRange = (startDate: DateTime, endDate: DateTime): string => {
+  return `${getMMMyyyy(startDate)} - ${getMMMyyyy(endDate)}`
+}
+
+type TravelPayClaimsDatePickerOption = {
+  label: string
+  value: TimeFrameType
+  a11yLabel: string
+  dates: TravelPayClaimsDateRange
+}
+
+const getPickerOptions = (t: TFunction): Array<TravelPayClaimsDatePickerOption> => {
+  const map = createTimeFrameDateRangeMap()
+
+  const pastThreeMonths = map[TimeFrameTypeConstants.PAST_THREE_MONTHS]
```

Comment:
Are these the same picker options as past appointments? Can we consolidate this entire picker options generation into a generic util?